src/lib/circuit_value.test.ts

```diff
@@ -0,0 +1,15 @@
+import { Bool, Circuit, isReady, shutdown, Int64 } from '../../dist/server';
+
+describe('circuit', () => {
```

Comment:
can you add tests expecting failures if there are more than one true and zero true?

Reply:
done. for some reason returning 0 for zero true seemed natural to me. but I can also throw in that case if you think that's better

Reply:
good point yeah 0 is fine, but can you add a test for the zero case succeeding too?